README.md

```diff
@@ -38,16 +38,34 @@ logzio/docker-collector-metrics
 | **LOGZIO_URL** | **Default**: listener.logz.io <br /> Logz.io listener URL to ship the metrics to. For more information on finding your account's region, see [Account region](https://docs.logz.io/user-guide/accounts/account-region.html).|
 | **LOGZIO_TYPE** | **Default**: `docker-collector-metrics` <br /> Logz.io type you'll use with this Docker. This is shown in your logs under the `type` field in Kibana. Logz.io applies parsing based on type. |
```

Comment:
@mirii1994 @yyyogev seems to me this is not needed anymore. Shall we remove it?

Reply:
Relevant as ever